packages/dx-react-chart/src/types/templates.series.types.ts

```diff
@@ -0,0 +1,110 @@
+import {
+  Scales, BuildAnimatedStyleGetterFn, TransformedPoint,
+} from '@devexpress/dx-chart-core';
+import { PureComputed } from '@devexpress/dx-core';
+
+type Path = PureComputed<[TransformedPoint[]], string>;
```

Comment:
It does not relate to computeds. Declare it as
```js
type Path = (points: ReadonlyArray<TransformedPoint>) => string
```
Maybe it should even be named `PathFn` instead of `Path`, though I am not sure of it.